packages/xrpl/test/integration/setup.ts

```diff
@@ -1,24 +1,132 @@
-import { Client, Wallet } from 'xrpl-local'
+import { AccountSet, Client, Payment, TrustSet, Wallet } from 'xrpl-local'
 
 import serverUrl from './serverUrl'
-import { fundAccount } from './utils'
+import { fundAccount, ledgerAccept } from './utils'
 
 export async function teardownClient(this: Mocha.Context): Promise<void> {
   this.client.removeAllListeners()
   this.client.disconnect()
 }
 
+// eslint-disable-next-line max-params -- need comments
+async function initIC(
```

Comment:
What is `IC`?

Reply:
issued currency

Reply:
Please spell it out

Reply:
I'd recommend using `IOU` or `Token` instead.